public/undefined.js

```diff
@@ -0,0 +1,18 @@
+webpackJsonp([8],{
```

Comment:
as I said in #12, delete this file

Reply:
deleted this file

Reply:
when i deleted this file, application don´t working (when i try logging to application, i got error)